types/dygraphs/index.d.ts

```diff
@@ -1235,7 +1235,7 @@ declare class Dygraph {
      * @param {number} width Width (in pixels)
      * @param {number} height Height (in pixels)
      */
-    resize(width: number, height: number): void;
+    resize(width?: number, height?: number): void;
```

Comment:
A test would be appreciated. The contract is really that `resize` can be called with zero or two parameters, but not one. You could model this with an overloaded declaration.

https://github.com/danvk/dygraphs/blob/e9155b2c442bf69396d5a456c885f938d38d35da/src/dygraph.js#L3177-L3181